### PR TITLE
Added accommodation information page.

### PR DESCRIPTION
**Pull request by SimonPStevens**

### CODE REVIEW

File: DDDEastAnglia/Views/Home/Accommodation.cshtml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    ViewBag.Title = "Accommodation";
+}
+
+<div class="page-header">
+    <h2>@ViewBag.Title</h2>
+</div>
+
+<p><strong>Please be aware that the DDD East Anglia weekend is a particularly busy time in Cambridge due to graduation. We recommend that you book your accommodation as soon as possible.</strong></p>
+
+<p>We would like to encourage attendees and speakers to use the brand new Travelodge on Newmarket road.</p>
```
Comment (adrianbanks) [inline DDDEastAnglia/Views/Home/Accommodation.cshtml:11]: "road" should be capitalised as it is part of the name of the road.

File: DDDEastAnglia/Views/Home/Accommodation.cshtml [hunk 2]
```diff
@@ -0,0 +1,122 @@
+﻿@{
+    ViewBag.Title = "Accommodation";
+}
+
+<div class="page-header">
```
Comment (adrianbanks) [inline DDDEastAnglia/Views/Home/Accommodation.cshtml:5]: This isn't styled like the other pages. This page has a horizontal rule below the heading.

Comment (SimonPStevens) [inline DDDEastAnglia/Views/Home/Accommodation.cshtml:5]: It's the same as the "About" page, although that seems to be the only one, so I suggest we remove the line from both.

File: DDDEastAnglia/Views/Home/Accommodation.cshtml [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    <h2>@ViewBag.Title</h2>
+</div>
+
+<p><strong>Please be aware that the DDD East Anglia weekend is a particularly busy time in Cambridge due to graduation. We recommend that you book your accommodation as soon as possible.</strong></p>
```
Comment (adrianbanks) [inline DDDEastAnglia/Views/Home/Accommodation.cshtml:9]: DDD East Anglia should be in the bolded green like elsewhere on the site. There is a helper method to do it in `HTMLExtensions`.

File: DDDEastAnglia/Views/Home/Accommodation.cshtml (hunk 2 above)
Comment (adrianbanks) [inline DDDEastAnglia/Views/Home/Accommodation.cshtml:5]: So it is. We should be consistent so probably best to remove it from both.

**Comment by adrianbanks** [comment]
In general ok. Is there any way to add some kind of separation between each hotel? The differing heights of each hotel's "block", combined with the alignment of the pictures makes it a bit hard to separate each hotel. Perhaps a feint grey border (like the green one) around the others, or a simple line between them?

**Comment by adrianbanks** [comment]
I've un-merged this as you are still making changes. When done, it can be merged again (either manually or by another pull request).

**Comment by SimonPStevens** [comment]
Recreated pull request #166 with fixes from comments.

**Pull request merged**